app/js/reducers/artifacts.js

```diff
@@ -1,34 +1,25 @@
-const initialState = [
-    {
-        name: 'table',
-        uuid: 'f16ca3d0-fe83-4b1e-8eea-7e35db3f6b0f',
-        type: 'FeatureTable[Frequency]'
-    },
-    {
-        name: 'phylogeny',
-        uuid: '908dece5-db23-4562-ad03-876bb5750145',
-        type: 'Phylogeny'
-    }
-];
+const initialState = [];
 
 const artifactsReducer = (state = initialState, action) => {
     switch (action.type) {
     case 'NEW_ARTIFACT': {
+        const { artifact } = action;
         const newState = [
             ...state,
-            action.artifact
+            artifact
         ];
         return newState;
     }
     case 'DELETE_ARTIFACT': {
-        // TODO, make it actually do something
-        const newState = state.filter(artifact => artifact.uuid !== action.uuid);
+        const filteredState = state.filter(a => a.uuid !== action.uuid);
+        const newState = [
+            ...filteredState
```

Comment:
Definitely! That's how it was, but this was another hidden leftover of the first input artifacts attempt where it had to be `state.artifacts.filter(...)` separately and then pieced together with the input artifacts.